### ISSUE TITLE
Reaper ATK screencast : Encode other multichannel formats

### ISSUE THREAD

**Issue by lossius** [ambisonictoolkit/ambisonictoolkit.github.io#26]
This tutorial will discuss the remaining multichannel encoders, except for 5_0 and 7_0:
- Quadrophonic
- Pantophonic 2D
- Periphonic 3D
- AtoB - to be discussed how this can be used to encode four channels of similar but non-correlated sound material in order to create a spatial texture - with further transforms in order to make it less ever-present.

I don't think that I'll include AmbiXtoB here, it's probably better to make a separate tutorial on conversions between FOA and AmbiX, and how to work with ATK for Reaper when creating VR content.

**Comment by joslloand** [comment]
@lossius 

Agreed, AmbiX in an ATK for VR would be the place for that discussion.
